lib/Default/WeightedRandom.class.inc

```diff
@@ -73,19 +73,14 @@ class WeightedRandom {
 		return $this->weighting;
 	}
 	
-	public function percentFailed($percent) {
-		$failed = mt_rand(0, 99) + $this->weighting >= $percent;
-		$this->adjustWeightByPercent(!$failed, $percent);
-		return $failed;
-	}
-	
-	public function adjustWeightByPercent($success, $percent) {
-		$percent = $percent / 100;
-		if ($success) {
-			$percent = -1 + $percent;
+	public function checkHit($weaponAccuracy) {
```

Comment:
This class was previously context-agnostic. Is it possible to rename the function and variables to maintain that? (Perhaps something like `checkSuccess`, `$percentChance`, and `$success` or whatever seems suitable.)